src/HtmlOptions/RequiredHtmlOptions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\HtmlOptions;
+
+use Yiisoft\Validator\Rule\Required;
+
+final class RequiredHtmlOptions implements HtmlOptionsProvider
+{
+    private Required $validator;
+
+    public function __construct(Required $validator)
```

Comment:
Why do we need constructor? Validator instance isn't used anywhere.

Reply:
The same as with `EmailHtmlOptions` above.